src/libcrun/cgroup.c

```diff
@@ -998,7 +1127,10 @@ int destroy_systemd_cgroup_scope (const char *scope, libcrun_error_t *err)
 #endif
 
 static int
-libcrun_cgroup_enter_internal (runtime_spec_schema_config_linux_resources *resources, int cgroup_mode, char **path, const char *cgroup_path, int manager, pid_t pid, const char *id, libcrun_error_t *err)
+libcrun_cgroup_enter_internal (runtime_spec_schema_config_linux_resources *resources,
+                               json_map_string_string *annotations arg_unused, int cgroup_mode,
```

Comment:
is the `arg_unused` here because of `HAVE_SYSTEMD` might be unset?

Reply:
yeah, it is unused only when not compiled with systemd